db/migrate/20190722114341_add_proxy_domains.rb

```diff
@@ -0,0 +1,20 @@
+class AddProxyDomains < ActiveRecord::Migration
+  def change
+    add_column :proxies, :staging_domain, :string
+    add_column :proxies, :production_domain, :string
+
+    add_index :proxies, [ :staging_domain, :production_domain ]
+
+    reversible do |dir|
+      dir.up do
+        Proxy.reset_column_information
+
+        Proxy.select(:id, :sandbox_endpoint, :endpoint).find_in_batches(batch_size: 1000) do |batch|
```

Comment:
If we don't need to address the batch for anything then we can simplify with `find_each`.

Reply:
Indeed. Use github suggested changes for it? :-D I'm wrapping up other things.

Reply:
> Use github suggested changes for it

I think that way the commits are not signed 🤔

Reply:
Merge commits are not signed either then.

Reply:
I think GH suggested changes doesn't work for multi-line changes.

Reply:
Anyway someone will have to work on this branch locally at some point and commit and push for the migrations 😄